apps/explorer/nuxt.config.ts

```diff
@@ -2,6 +2,10 @@ import { defineNuxtConfig } from 'nuxt/config'
 
 export default defineNuxtConfig({
   modules: ['@nuxtjs/tailwindcss', '@nuxtjs/color-mode', 'nuxt-graphql-client'],
+  components: {
+    global: true,
+    dirs: ['~/components']
+  },
```

Comment:
Why is this modification needed? I think in Nuxt 3 all components are global by default.

Reply:
It isn't or else it doesn't let you change the variable state in the template